src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -1406,6 +1429,10 @@ VideoDecoder::~VideoDecoder() {
   }
 }
 
+FrameDims getHeightAndWidthFromResizedAVFrame(const AVFrame& resizedAVFrame) {
```

Comment:
Because the function itself has no opinion on what kind of frame it is (resized or not), I think the name should just be `getHeightAndWidthFromAVFrame()`.